changelogs/fragments/5370-mh-cmdmixin-deprecation.yaml

```diff
@@ -0,0 +1,5 @@
+deprecated_features:
+  - CmdMixin module utils - deprecated in favor of the ``CmdRunner`` module util (https://github.com/ansible-collections/community.general/pull/5370).
+  - CmdModuleHelper module utils - deprecated in favor of the ``CmdRunner`` module util (https://github.com/ansible-collections/community.general/pull/5370).
+  - CmdStateModuleHelper module utils - deprecated in favor of the ``CmdRunner`` module util (https://github.com/ansible-collections/community.general/pull/5370).
```

Comment:
I would also add comments in `CmdModuleHelper` and `CmdStateModuleHelper` that these are deprecated, so you can also figure this out quickly from looking at the sources.